test/common/index.js

```diff
@@ -102,46 +102,6 @@ function expectsError (validator, exact) {
   }, exact)
 }
 
-if (typeof AbortSignal.timeout !== 'function') {
-  // `AbortSignal.timeout` is not available on Node.js 14.x, we need to polyfill
-  // it because some tests are using it. End-users don't need to it.
-
-  class AbortError extends Error {
-    constructor (message = 'The operation was aborted', options = undefined) {
-      super(message, options)
-      this.code = 23
-    }
-  }
-
-  AbortSignal.timeout = function timeout (delay) {
-    const ac = new AbortController()
-    setTimeout(() => ac.abort(new AbortError(
-      'The operation was aborted due to timeout')), delay).unref()
-    return ac.signal
-  }
-}
-
-if (process.version.startsWith('v14.') || process.version.startsWith('v16.')) {
-  // Implementation of AbortSignal and AbortController differ slightly with the
-  // v18.x one, creating some difference on the TAP output which makes the tests
-  // fail. We are overriding the built-ins to make the test pass, however that's
-  // not necessary to do for the library to work (i.e. end-users don't need it).
-
-  const defaultAbortError = new Error('This operation was aborted')
-  defaultAbortError.code = 20
```

Comment:
this might still be needed

Reply:
No, because we no longer run the `test/message`, and keeping it breaks Node.js v14.x without the CLI flag.